exampleSite/hugo.yaml

```diff
@@ -133,6 +133,7 @@ params:
 
   editURL:
     enable: true
+    where: "toc" # single | toc
```

Comment:
A nitpick, but I'd prefer a more descriptive name for this field, such as `location` or `placement`.